src/main/java/com/school/project/model/entities/Group.java

```diff
@@ -1,13 +1,34 @@
 package com.school.project.model.entities;
 
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
 import java.util.Date;
 import java.util.List;
-
+@Entity
+@Table(name = "GROUP")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class Group {
-
+    @Id
+    @Column(name="ID", unique = true,nullable = false)
```

Comment:
Pls use: The auto-formatting shortcut in IntelliJ

For windows Ctrl+Alt+L.
For ubuntu Ctrl+Alt+windows+L.
For Mac Command+Option+L.